### PR TITLE
plugin rename

### LINKED ISSUE #5: Name change - mediaWithBenefits > mediaGallery
Change the name of the plugin to something a little more descriptive

### PR DESCRIPTION
**Pull request by guywillis**
resolves https://github.com/cgkineo/adapt-mediaWithBenefits/issues/5

* Rename plugin from Media with Benefits to Media Gallery

### CODE REVIEW

File: js/mediaGalleryView.js [hunk 1]
```diff
@@ -143,21 +143,24 @@ define([
...
+      $transcriptBodyContainer.stop(true, true).slideDown(() => {
         $(window).resize();
       }).addClass('inline-transcript-open');
       $button.attr('aria-expanded', true);
       $buttonText.html(this.model.get('_transcript').inlineTranscriptCloseButton);
 
-      this.checkCompletionByTranscript();
+      if (this.model.get('_transcript')._setCompletionOnView !== false) {
+        this.setCompletionStatus();
+      }
```
Comment (moloko) [inline js/mediaGalleryView.js]: was this change deliberate?

File: bower.json [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
+  "name": "adapt-mediaGallery",
+  "version": "2.0.0",
   "framework": ">=5.3",
-  "homepage": "https://github.com/cgkineo/adapt-mediaWithBenefits",
-  "issues": "https://github.com/cgkineo/adapt-mediaWithBenefits/issues/ ",
-  "component": "mediawithbenefits",
-  "displayName": "Media with benefits",
+  "homepage": "https://github.com/cgkineo/adapt-mediaGallery",
+  "issues": "https://github.com/cgkineo/adapt-mediaGallery/issues/ ",
+  "component": "mediaGallery",
```
Comment (moloko) [inline bower.json]: not 100% sure about this but I _think_ this needs to be lowercase to match the component registration/name

File: bower.json [hunk 3]
```diff
@@ -1,13 +1,13 @@
 {
-  "name": "adapt-mediaWithBenefits",
-  "version": "1.0.0",
+  "name": "adapt-mediaGallery",
+  "version": "2.0.0",
```
Comment (moloko) [inline bower.json]: version v1.0.0 hasn't been released so there's no particular need to bump this to v2.0.0

File: js/mediaGalleryView.js (hunk 1 above)
Comment (guywillis) [reply js/mediaGalleryView.js]: It is, to mirror the current media implementation of the inline transcript.

**Review by oliverfoster — APPROVED** [review]

**Review by kirsty-hames — APPROVED** [review]
👀

**Review by oliverfoster — APPROVED** [review]
